src/tests/LoginTest.java

```diff
@@ -0,0 +1,44 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class LoginTest extends BaseTest{
+
+    @BeforeMethod
+    public void beforeTest(){
+        HomePage home = new HomePage(driver);
+        home.openHomePage()
+            .goToSignPage();
+    }
+
+    @Test(groups = {"Smoke", "Regression"})
+    public void pageElement(){
```

Comment:
Test methods name should discribe what we have to do and what we have to assert, so please rename tests method name